projectional/src/main/java/jetbrains/jetpad/projectional/cell/BaseProjectionalSynchronizer.java

```diff
@@ -66,6 +70,8 @@
   private DeleteHandler myDeleteHandler = DeleteHandler.EMPTY;
   private ContentKind<SourceItemT> myItemKind;
   private Function<SourceItemT, SourceItemT> myCloner;
+  private ListMap<ContentKind, ContentToItemMapper> myCompatibleContentKinds = new ListMap<>();
+  private ListMap<ContentKind, ContentToItemsListMapper> myContentKindsCompatibleToItemsList = new ListMap<>();
```

Comment:
We can't name variables "someContentKindWhichCanBeMappedToListSourceItem" or myContentKindsCompatibleToItemsList which I didn't understand at all at first. You have two close variables myCompatibleContentKinds and myContentKindsCompatibleToItemsList and they have quite different names and one can they have different types (something that ends in "List" is usually a list).